rviz2/scripts/rviz1_to_rviz2.py

```diff
@@ -0,0 +1,488 @@
+#!/usr/bin/env python3
+
+import argparse
+import copy
+import sys
+import warnings
+import yaml
+
+
+def del_maybe(dictionary, key):
+    if key in dictionary:
+        del dictionary[key]
+
+
+def migrate_panels(panels):
+    panels_rviz2 = []
+
+    for panel_dict in panels:
+        if 'Class' not in panel_dict:
+            print('Uknown panel format, skipping:' + repr(panel_dict), file=sys.stderr)
+            continue
+
+        if 'rviz/Displays' == panel_dict['Class']:
+            panels_rviz2.append(migrate_panel_displays(panel_dict))
+        elif 'rviz/Selection' == panel_dict['Class']:
+            panels_rviz2.append(migrate_panel_selection(panel_dict))
+        elif 'rviz/Tool Properties' == panel_dict['Class']:
+            panels_rviz2.append(migrate_panel_tool_properties(panel_dict))
+        elif 'rviz/Views' == panel_dict['Class']:
+            panels_rviz2.append(migrate_panel_views(panel_dict))
+        elif 'rviz/Time' == panel_dict['Class']:
+            panels_rviz2.append(migrate_panel_time(panel_dict))
```

Comment:
Should we add an `else` block for unsupported panels (not sure if that's possible)?

Reply:
`else` block that warns about skipping an uknown panel in 520277bf512eb40774